.github/workflows/_linux_accelerate.yml

```diff
@@ -0,0 +1,156 @@
+name: Linux Accelerate Test
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '.github/scripts/spec.py'
+      - '.github/actions/print-environment/action.yml'
+      - '.github/workflows/_linux_accelerate.yml'
+  workflow_dispatch:
+    inputs:
+      pytorch:
+        required: false
+        type: string
+        default: 'nightly'
+        description: Pytorch branch/commit
+      python:
+        required: false
+        type: string
+        default: '3.10'
+        description: Python version
+      runner:
+        required: true
+        type: string
+        default: 'linux.idc.xpu'
+        description: Runner label
+      nightly_whl:
+        required: false
+        type: string
+        default: ''
+        description: Pytorch nightly wheel version
+      accelerate:
+        required: false
+        type: string
+        default: 'v1.2.1'
+        description: Accelerate version
+
+permissions: read-all
+
+jobs:
+  Torch-XPU-Accelerate-Tests:
+    runs-on: ${{ inputs.runner != '' && inputs.runner || 'linux.idc.xpu' }}
+    env:
+      CONDA_ENV_NAME: 'huggingface_accelerate_test'
+      WORK_DIR: 'accelerate'
+      NEOReadDebugKeys: 0
+      DisableScratchPages: 0
+      accelerate: ${{ inputs.accelerate != '' && inputs.accelerate || 'v1.2.1' }}
+      python: ${{ inputs.python != '' && inputs.python || '3.10' }}
+      pytorch: ${{ inputs.pytorch != '' && inputs.pytorch || 'nightly' }}
+      PYTORCH_DEBUG_XPU_FALLBACK: 1
+      ZE_AFFINITY_MASK: 0
+      PARSE_JUNIT: ${{ github.workspace }}/torch-xpu-ops/.github/scripts/parse-junitxml.py
+    steps:
+      - name: Checkout torch-xpu-ops
```

Comment:
Seems we don't need checkout torch-xpu-ops

Reply:
We need it. Workflow uses:
* .github/scripts/parse-junitxml.py
* .github/actions/print-environment/action.yml

from torch-xpu-ops repo. However, I forgot to add .py script to the dependencies of this workflow and I also I forgot to drop `spec.py` which is Transformers specific - I will fix that.

Reply:
> However, I forgot to add .py script to the dependencies of this workflow and I also I forgot to drop spec.py which is Transformers specific - I will fix that.

Done.